recipes/hdf4/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib
```

Comment:
No worries. I am the same. Fortunately GitHub seems to keep track of these. Don't know when runs GC though.